index.html

```diff
@@ -248,9 +221,16 @@
                 game.load.audio('dissolve', 'assets/audio/dissolve.mp3')
                 game.load.audio('move', 'assets/audio/move.mp3')
 
+                game.load.audio('bgm', 'assets/audio/bgm.mp3')                
+                game.load.audio('lightning', 'assets/audio/lightning.mp3')
+                game.load.audio('victory', 'assets/audio/victory.mp3')
+                game.load.audio('resolve', 'assets/audio/resolve.mp3')
+                game.load.audio('profit', 'assets/audio/profit.mp3')
+                game.load.audio('warning', 'assets/audio/warning.mp3')
+
                 game.load.spritesheet('vortex', 'assets/sprites/vortexSprite.png', 64, 64);
                 game.load.spritesheet('vortexRed', 'assets/sprites/vortexRed.png', 64, 64);
-                gameInfo.load.spritesheet('attack', 'assets/sprites/attack.png', 522, 522);
```

Comment:
Will the attack sprites still work if we load them into `game`? We're using them in `gameInfo` [here](https://github.com/h223chen/match3-hackathon/blob/c4abfa42944a92b4307d8ada80810b17a2b4300b/index.html#L1020) and [here](https://github.com/h223chen/match3-hackathon/blob/c4abfa42944a92b4307d8ada80810b17a2b4300b/index.html#L1041)